internal/db/read_writer_ext_test.go

```diff
@@ -16,8 +16,8 @@ import (
 
 func TestDb_Create_OnConflict(t *testing.T) {
 	ctx := context.Background()
-	wrapper := db.TestWrapper(t)
 	conn, _ := db.TestSetup(t, "postgres")
+	wrapper := db.TestDBWrapper(t, conn, "oplog")
```

Comment:
A lot of tests that previously were happy enough to use an in-memory wrapper now require a full DB wrapper, since oplog entries require a key_id that exists.